readme.md

```diff
@@ -127,6 +128,28 @@ Directories are recursive by default. Files in directories named `fixtures` and
 
 *WARNING: NON-STANDARD BEHAVIOR:* The AVA CLI will always try to find and use your projects local install of AVA. This is true even when you run the global `ava` command. This non-standard behavior solves an important [issue](https://github.com/sindresorhus/ava/issues/157), and should have no impact on everyday use.
 
+## Configuration
+
+All of the CLI options can be configured in the `ava` section of your `package.json`:
+
+```json
+{
+  "ava": {
+    "files": [
+      "my-test-folder/*.js",
+      "!**/not-this-file.js"
+    ],
+    "fail-fast": true,
+    "serial": true,
+    "tap": true,
+    "verbose": true,
+    "require": ["babel-core/register", "coffee-script/register"]
+  }
+}
+```
+
+Arguments passed to the CLI will always take precedence over the configuration in `package.json`.
+
```

Comment:
Maybe mention the benefit of specifying config here instead of the CLI is that it will be used when you run it with `$ ava`.